tutoraspects/asset_command_helpers.py

```diff
@@ -137,10 +137,10 @@ def omit_templated_vars(self, content: dict, existing: dict):
                     else:
                         content[key] = existing[key]
 
-            if isinstance(existing[key], dict):
```

Comment:
Why was this change needed?

Reply:
Because it was checking the existing value and assuming the new value was also a `dict`, however it was not the case